source/test_env.cpp

```diff
@@ -109,19 +112,19 @@ void GREENTEA_TESTCASE_FINISH(const char *test_case_name, const size_t passes, c
  */
 #ifdef YOTTA_CFG_DEBUG_OPTIONS_COVERAGE
 extern "C" void __gcov_flush(void);
-bool coverage_report = false;
+extern bool coverage_report;
```

Comment:
where is this defined? Is it in mbed-drivers? I did not spot it in the dependencies